src/main/webapp/app/entities/participation/participation-websocket.service.ts

```diff
@@ -209,9 +192,11 @@ export class ParticipationWebsocketService {
      *
      * If there is no observable for the participation a new one will be created.
      *
-     * @param participationId ID of the participation that the new results should belong to
+     * @param participationId Id of Participation of which result to subscribe to
+     * @param exercise Exercise to which the Participation belongs
      */
-    subscribeForLatestResultOfParticipation(participationId: number): BehaviorSubject<Result> {
+    public subscribeForLatestResultOfParticipation(participationId: number): BehaviorSubject<Result> {
+        this.createResultWSConnectionIfNotExisting(participationId).subscribe();
```

Comment:
This is the only thing that needs to be done to enable result subscription without providing a participation.